pkg/sql/pgwire/testdata/pgtest/float

```diff
@@ -0,0 +1,56 @@
+send
+Query {"String": "SELECT (1/3.0)::float4, (1/3.0)::float8"}
+----
+
+until
+ReadyForQuery
+----
+{"Type":"RowDescription","Fields":[{"Name":"float4","TableOID":0,"TableAttributeNumber":0,"DataTypeOID":700,"DataTypeSize":4,"TypeModifier":-1,"Format":0},{"Name":"float8","TableOID":0,"TableAttributeNumber":0,"DataTypeOID":701,"DataTypeSize":8,"TypeModifier":-1,"Format":0}]}
+{"Type":"DataRow","Values":[{"text":"0.33333334"},{"text":"0.3333333333333333"}]}
```

Comment:
ooh we round up to 4? is this consistent when running this with PG?

Reply:
yep,  the expected result was obtained by running against PG